bin/redshift/run_mvn_tests_on_aws.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# Runs Redshift connector tests.
+#
+# Run setup_aws_redshift.sh as a prerequisite for creating the Redshift instance and viewing
+# required environment variables.
+#
+# Usage:
+# run_mvn_tests_on_aws.sh
+
+set -xeuo pipefail
+
+ROOT_DIR="$(git rev-parse --show-toplevel)"
+cd "$ROOT_DIR"
+
+trap terminate INT TERM EXIT
+
+suite_exit_code=0
+
+MAVEN_OPTS="$MAVEN_INSTALL_OPTS" ./mvnw -s .github/settings.xml\
```

Comment:
do we have the `.github/settings.xml` file?

instead of this use `${MAVEN}` (defined by ci.yml)